doc_templates/README.md

```diff
@@ -1,34 +1,27 @@
 # CLI documentation templates
 
-These templates provide extra information for the automatically generated CLI
-documentation.
+These templates provide extra information for the automatically generated CLI documentation.
```

Comment:
Don't really like that formatting was changed here. It certainly makes the review significantly more difficult.

Just for reference, the previous column width was 80, which I think is the best choice for reading plain text.

Reply:
This was a tough change to make because I also had previously been using a fixed line length as well...albeit a larger one than 80. That is kind of the point...there is no enforced standard for markdown and it would/could change between authors. Here are some reasons why hard wrapping markdown text was removed in favor of (possibly) longer lines and soft wrapping:

* Hard wrapping requires more author effort, especially when modifying existing text
* Lines with link/image reference definitions are harder to split without breaking the URL
* The markdown source does not look like the rendered output, in which there are no line breaks
* Manual line breaking can make the Markdown more readable than the rendered output, which is bad because it gives a false sense of readability encouraging less readable long paragraphs
* Markdown format characters count against the line limit even if they wouldn't in rendered form
* Individual developers can choose to set their editors and diff tools to view the source/diffs the way they prefer
* GitHub's web interface as well as many desktop Git tools feature substring change highlighting within lines and side-by-side or similar version comparison so there is no need for artificial line breaks
* A quick survey of other popular Docusaurus powered sites revealed they do not enforce hard wrapping
  * [Docusaurus itself](https://github.com/facebook/docusaurus/blob/main/website/docs/deployment.mdx)
  * [Redis](https://github.com/redis-developer/redis-developer.github.io/blob/master/docs/create/redis-stack/index-redis-stack.mdx)
  * [React](https://github.com/facebook/react-native-website/blob/main/docs/introduction.md?plain=1)
  * [Supabase](https://github.com/supabase/supabase/blob/master/apps/docs/pages/guides/auth.mdx) (although they seem to hard break on sentence boundaries)

Reply:
I have not used it personally, and I'm not sure it could be helpful to auto-enforce Markdown formatting, but I think we could consider checking out [editorconfig](https://editorconfig.org/).

I have checked over time for Markdown auto-formatting solutions but I have never found anything that really fit the bill in an acceptable way. It would definitely be a huge value add for a case like this, though.

Reply:
The conversation has derailed from the original comment.

The debate between hard breaks and soft breaks can wait. This PR is about the move to Docusaurus. Significant and unrelated formatting changes belong in a separate PR.

Reply:
Agreed...it should have been done separately. There was an intermediate commit, after initializing the Docusaurus site, where the `docs` directory was moved to `website/docs`. That is when the formatting changes were made b/c it was expected that the diff would not look pretty anyway. Later, it was discovered that the `docs` directory could live elsewhere and so it was changed back...but the formatting kept.

I'll create a separate issue to discuss markdown standardization...for this repo and ideally as a company standard. The formatting changes made here will be reverted in a best effort approach.

Reply:
The changes have been reverted and https://github.com/phylum-dev/cli/issues/1154 was created.
The https://cli.phylum.io site was deployed with these latest changes for assisting with the review.